src/pages/_app.tsx

```diff
@@ -132,6 +134,25 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
       {getLayout(<Component {...pageProps} />)}
 
       <Toaster />
+
+      <div
+        id="idos_container"
+        style={
+          !router.route.includes('settings')
+            ? ({
+                position: 'absolute',
+                top: 0,
+                left: 0,
+                width: 0,
+                height: 0,
+                margin: 0,
+                padding: 0,
+                opacity: 0,
+                overflow: 'hidden',
+              } as React.CSSProperties)
+            : undefined
+        }
+      />
```

Comment:
Nice solution! We just need to update this logic so that when it is the settings page, we render the div like we'd expect with the previous CSS. Right now the div is invisible when you are on the settings page:

<img width="1468" alt="Screen Shot 2023-11-03 at 3 34 56 PM" src="https://github.com/near/near-discovery/assets/1475067/71fce747-bc07-4878-9eec-03ef21696b6e">

Maybe we can do this by applying a CSS class and we can move these styles to `globals.css`?

Reply:
Also, just incase we have other routes with the word `settings`, let's adjust to use this logic:

```js
router.route.startsWith('/settings')
```

The previous code would also match something like `/crazy/route-settings`